models/user.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+type User struct {
+	Base
+
+	Name              string
+	Email             string `orm:"unique"`
+	HashedPassword string
```

Comment:
Alignment issue for the data type 

```suggestion
	HashedPassword    string
```